server/routes.js

```diff
@@ -120,8 +121,8 @@ module.exports = (app) => {
   /**
    * Groups.
    */
+  app.post('/groups', ifParam('flow', 'github'), aN.authenticateAppByApiKey, aN.parseJwtNoExpiryCheck, aN.checkJwtExpiry, required('payload'), groups.createFromGithub); // Create a group from a github repo
   app.post('/groups', aN.authenticateUserByJwt(), required('group'), groups.create); // Create a group. Option `role` to assign the caller directly (default to null).
```

Comment:
I did that first and then realized that it makes a ton of tests fail and we'll have to add the field everywhere. Maybe still worth doing as a separate PR.